content/en/docs/concepts/security/service-accounts.md

```diff
@@ -0,0 +1,241 @@
+   might include service account definitions for the system's components. The
+   lightweight nature of service accounts and the namespaced identities make
+   the configurations portable.
+
+Service accounts are different from user accounts, which are authenticated
+human users in the cluster. By default, user accounts don't exist in the Kubernetes API server; instead, the API server treats user identities as opaque
+data. You can authenticate as a user account using multiple methods. Some
+Kubernetes distributions might add custom extension APIs to represent user
+accounts in the API server.
+
+{{< table caption="Comparison between service accounts and users" >}}
+
+| Description | ServiceAccount | User or group |
+| --- | --- | --- |
+| Location | Kubernetes API (ServiceAccount object) | External |
+| Scope | Namespace | Cluster |
+| Access control | Kubernetes RBAC or other [authorization mechanisms](/docs/reference/access-authn-authz/authorization/#authorization-modules) | Kubernetes RBAC or other identity and access management mechanisms |
+| Intended use | Pods | People |
+
+{{< /table >}}
+
+### Default service accounts {#default-service-accounts}
+
+When you create a cluster, Kubernetes automatically creates a ServiceAccount
+object named `default` for every namespace in your cluster. The `default`
+service accounts in each namespace get no permissions by default other than the
+[default API discovery permissions](/docs/reference/access-authn-authz/rbac/#service-account-permissions)
+that Kubernetes grants to all authenticated accounts. If you delete the
+`default` ServiceAccount object in a namespace, the
+{{<glossary_tooltip text="control plane" term_id="control-plane">}}
+replaces it with a new one.
+
+If you deploy a Pod in a namespace, and you don't manually assign a
+ServiceAccount object to the Pod, Kubernetes assigns the `default` service
+account for that namespace to the Pod and provides the credentials for that
+ServiceAccount to the Pod. To prevent Kubernetes from automatically providing
+credentials for the `default` ServiceAccount, set the
+`automountServiceAccountToken` field in your Pod specification to `false`.
+
+## Use cases for Kubernetes service accounts {#use-cases}
+
+As a general guideline, you can use service accounts to provide identities in
+the following scenarios:
+
+* Your Pods need to communicate with the Kubernetes API server, for example in
+  situations such as the following:
+  *  Providing read-only access to sensitive information stored in Secrets.
+  *  Granting [cross-namespace access](#cross-namespace), such as allowing a
+     Pod in namespace `example` to read, list, and watch for Lease objects in
+     the `kube-node-lease` namespace.
+* Your Pods need to communicate with an external service. For example, a
+  workload Pod requires an identity for a commercially available cloud API,
+  and the commercial provider allows configuring a suitable trust relationship.
```

Comment:
This might be a good place to refer to SA token audiences

Reply:
OK, but IMO not having it here shouldn't block approval. The perfect is the enemy of the published.

Reply:
I understand the idea but we currently dont have a lot of info in our docs about audiences...I'd like to see better info to link to from here, expanding on https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#serviceaccount-token-volume-projection

WDYT?

Reply:
I think we might often be relying on the readers knowledge of how audiences work, as described in https://openid.net/specs/openid-connect-core-1_0.html#IDTokenValidation

I added a proposal in https://github.com/kubernetes/website/pull/38289/files#r1099997580 which I think might suffice for this new documentation

Reply:
Done